### PR TITLE
meta_knowledge_graph attributes

### PR DESCRIPTION
**Pull request by vdancik**
addresses #257

### CODE REVIEW

File: TranslatorReasonerAPI.yaml [hunk 1]
```diff
@@ -737,11 +743,51 @@ components:
...
+          description: >-
+            Source of an attribute provided by this TRAPI web service.
+          example: infores:chembl
+        original_attribute_names:
+          type: array
+          items:
+            type: string
+          description: >-
+            Names of an the attribute as provided by the source.
+        costraint_use:
```
Comment (CaseyTa) [inline TranslatorReasonerAPI.yaml:777]: typo: `constraint_use`

File: TranslatorReasonerAPI.yaml [hunk 2]
```diff
@@ -737,11 +743,51 @@ components:
...
+          type: boolean
+          description: >-
+            Indicates whether this attribute can be used as a query
+            constraint.
+          example: true
+        constraint_name:
```
Comment (CaseyTa) [inline TranslatorReasonerAPI.yaml:783]: I think the constraint name is supposed to be the same original_attribute_name, so this may be redundant.

Comment (vdancik) [reply TranslatorReasonerAPI.yaml:783]: Not necessarily as the same attribute can have different original names from different sources as shown in an example below.

**Comment by vdancik** [comment]
Example meta-attribute JSON:

```
{
    "attributes": [
        {
            "attribute_type_id": "biolink:p_value",
            "attribute_source": "infores:ctrp",
            "original_attribute_names": [
                "p-value"
            ],
            "constraint_use": true,
            "constraint_name": "p-value"
        },
        {
            "attribute_type_id": "biolink:p_value",
            "attribute_source": "infores:chembl",
            "original_attribute_names": [
                "pValue"
            ],
            "constraint_use": true,
            "constraint_name": "p-value"
        },
        {
            "attribute_type_id": "biolink:publication",
            "attribute_source": "infores:chembl",
            "original_attribute_names": [
                "PubMed",
                "reference"
            ],
            "constraint_use": false
        }
    ]
}
```

**Review by edeutsch — APPROVED** [review]

**Review by rtroper — APPROVED** [review]

**Comment by CaseyTa** [comment]
I still have a few questions:

1. The schema only defines `attribute_type_id` as required, but are we expected to provide as much detail as possible for all attributes (similar to the example above)? 
2. How does this work for the ARAs or KPs that may be accessing dynamic information (e.g., the Knowledge Collaboratory)?  
3. Does the presence of `constraint_name` imply that clients should specify that `constraint_name` exactly? Or would it still essentially be ignored by the service in favor of the `attribute_type_id`?

**Review by marcdubybroad — APPROVED** [review]

**Review by CaseyTa — APPROVED** [review]

**Pull request merged**